test/e2e/tests/request-queuing/multi-dapp-sendTx-revokePermission.spec.js

```diff
@@ -48,14 +45,11 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks revok
         await openDapp(driver, undefined, DAPP_URL);
 
         // Connect to dapp 1
-        await driver.findClickableElement({ text: 'Connect', tag: 'button' });
-        await driver.clickElement('#connectButton');
```

Comment:
clickElement already ensures that the button is clickable, so we can remove the extra step